geth/common/types.go

```diff
@@ -171,9 +173,21 @@ type TxQueueManager interface {
 	DiscardTransactions(ids string) map[string]RawDiscardTransactionResult
 }
 
+// JailExecutor defines an interface which exposes method to be executed
+// against a Jail vm.
+type JailExecutor interface {
+	EvalExec(string) error
+	Run(string) (otto.Value, error)
+	Exec(string) (otto.Value, error)
```

Comment:
The bug question is if we wish ro run everything on the loop? If we dont mind such then we can remove it.